src/js/components/award/details/DetailsTabBar.jsx

```diff
@@ -8,39 +8,40 @@ import React from 'react';
 import DetailsTabItem from './DetailsTabItem';
 
 const propTypes = {
-    tabs: React.PropTypes.array,
     activeTab: React.PropTypes.string,
-    clickTab: React.PropTypes.func
+    clickTab: React.PropTypes.func,
+    type: React.PropTypes.string
 };
 
-const defaultProps = {
-    tabs: [
-        {
-            label: 'Transaction History',
-            code: 'transaction',
-            disabled: false
-        },
-        {
-            label: 'Sub-Awards',
-            code: 'subaward',
-            disabled: false
-        },
-        {
-            label: 'Financial System Details',
-            code: 'financial',
-            disabled: false
-        },
-        {
-            label: 'Additional Details',
-            code: 'additional',
-            disabled: false
-        }
-    ]
-};
+const tabOptions = [
+    {
+        label: 'Transaction History',
+        code: 'transaction',
+        disabled: false
+    },
+    {
+        label: 'Sub-Awards',
```

Comment:
Why is subawards disabled? Subawards is implemented in dev, please make sure you are up-to-date and have reconciled the changes. Additionally, check to see which award types should have subawards and display as appropriate.